parlai/core/build_data.py

```diff
@@ -293,8 +294,37 @@
     if delete:
         PathManager.rm(fullpath)
 
+def ungzip(path, fname, deleteGZip=True):
```

Comment:
We already have untar

Reply:
Yes, I tried it with untar function (and its unzip) but they couldn't unzip '.gz'. files. But I added this function under untar. It calls this if the suffix is '.gz'.

Reply:
Is the file gz but not tar.gz?

Reply:
It is just '.jsonl.gz'